tests/boards/tenstorrent/tt_blackhole/dma/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_ZTEST=y
+
+# enable DMA subsystem
+CONFIG_DMA=y
+
+# enable NOC DMA driver
+CONFIG_DMA_TT_NOC_MAX_CHANNELS=8
```

Comment:
This should be a devicetree property, since different instances might have a different number of channels.

I think there is a devicetree property already defined for it in `dma-controller.yaml`.